doc/riscv_trace.rst

```diff
@@ -0,0 +1,69 @@
+.. _riscv_trace:                                                                                         
+
+   :name: riscv-trace-spec
+   :align: center
+
+RISC-V Trace Specs Interface
+============================
+
+Ibex implements a trace interface compatible with the riscv-trace-spec Version 1.0 issued in March 20, 2020. 
+This interface currently exports needed internal signal as ibex_core outputs to be used to encode the trace packets to be logged.
```

Comment:
Most of the time, you can remove words such as "currently" from the documentation. Everything can change, it is implied that the documentation refers to the current state of the implementation.

```suggestion
This interface exports needed internal signals as ibex_core outputs to be used to encode the trace packets to be logged.
```